src/Symfony/Component/Translation/Bridge/Loco/Tests/LocoProviderTest.php

```diff
@@ -1170,4 +1170,34 @@ public static function getResponsesForReadWithLastModified(): \Generator
             yield [$locales, $domains, $responseContents, $lastModifieds, $expectedTranslatorBag];
         }
     }
+
+    public function testReadWithRestrictToStatus()
+    {
+        $loader = $this->getLoader();
+
+        $loader
+            ->expects($this->once())
+            ->method('load')
+            ->willReturn($this->createMock(MessageCatalogue::class));
+
+        $provider = self::createProvider(
+            new MockHttpClient([
+                function (string $method, string $url, array $options = []): ResponseInterface {
+                    $this->assertSame('GET', $method);
+                    $this->assertSame('https://localise.biz/api/export/locale/de.xlf?filter=messages&status=translated%2Cprovisional', $url);
+                    $this->assertSame(['filter' => 'messages', 'status' => 'translated,provisional'], $options['query']);
+
+                    return new MockResponse('[{"id":"messages__a"}]');
```

Comment:
Please use JsonMockResponse here

Reply:
I have removed the returned json data. Wasnt relevant for the test.